cms/templates/partials/hero.html

```diff
@@ -1,4 +1,22 @@
-{% load wagtailembeds_tags %}
+{% load wagtailembeds_tags wagtailimages_tags %}
+
+{% block extrahead %}
```

Comment:
Not that that it's actually being used as of right now, however you're missing a `{{ block.super }}` here which would overwrite the `extrahead` block rather than append to it.

Reply:
I got an error saying that this was a base template so `block.super` was not allowed, I think. 
![Screenshot from 2019-06-24 11-21-39](https://user-images.githubusercontent.com/863262/60031238-5baccc00-9672-11e9-8b2b-504fc9b5dd5f.png)